olive/passes/onnx/conversion.py

```diff
@@ -195,22 +219,95 @@ def _convert_model_on_device(
         # Reset to CPU so the resource consumed on GPU could be free.
         if device != "cpu":
             pytorch_model.to("cpu")
+
         if torch.cuda.is_available():
             torch.cuda.empty_cache()
-        # save the model to the output path and return the model
-        return model_proto_to_olive_model(onnx_model, output_model_path, config)
 
-
-class DeviceSpecificOnnxConversion(OnnxConversion):
-    """Convert a PyTorch model to ONNX model using torch.onnx.export by using specific hardware device."""
+        return onnx_model
 
     @staticmethod
-    def is_accelerator_agnostic(accelerator_spec: AcceleratorSpec) -> bool:
-        return False
+    def _export_ranked_model(params):
+        pass_config, data_root, model_config, world_size, device, local_rank, output_dirpath = params
+
+        import os
```

Comment:
Prefer importing on the top of the module in the global scope

Reply:
Done!